tar/bsdtar.c

```diff
@@ -312,6 +344,16 @@ main(int argc, char **argv)
 				lafe_errc(1, 0,
 				    "Couldn't exclude %s\n", bsdtar->argument);
 			break;
+		case OPTION_EXCLUDE_VCS: /* GNU tar */
+			{
+				for(int i=0; vcs_files[i]; i++) {
```

Comment:
As reported in the Travis CI failure, we build with C90, so cannot use C99 shorthand.   This should be
```
    int i;
    for (i = 0; vcs_files[i]; i++) {
```